views/pages/singleEpisode.ejs

```diff
@@ -0,0 +1,12 @@
+<%- include("../partials/nav.ejs") %>
+
+<% for (let episode of gameofThronesEpisodes) { %>
+    <div class="card">
+        <h3>Title: <%= episode.name %> <%= makeEpisodeCode(episode)%></h3>
+        <img src="<%= episode.image.original %>" alt="Episode Cover Image"/>
```

Comment:
Some tv shows have episodes whose image property is set to null, so this will cause a crash when given those (e.g. episodes of the simpsons not yet broadcast, whose screenshots aren't available yet).